src/model/rest/nn_seeker_paservice.py

```diff
@@ -33,6 +33,7 @@ def get_k_NN(self, item: ItemDto, k, nn_filter) -> tuple[list, list, str]:
 
         params = {
             "configuration": self.__configuration_c2c,
+            "similarityType": model_props['param_model_type'],
```

Comment:
Hey, wenn ich es richtig verstehe würden wir den Parameter `param_model_type` dann verschieden handhaben in C2C vs U2C - C2C ist es immer `similarityType ` und U2C `modelType ` korrekt?

Reply:
Das stimmt, wir befüllen damit zwei unterschiedliche Dinge, ich fand deren Auswirkung aber ähnlich genug, um zu rechtfertigen, die Werte in dieselbe Prop in unserer Config zu schreiben